Delete Button, Username login, Updated Scrapeer, Security Fixes

AzukaChan opened the pull request:
I completely deleted my fork and started again, now there are 2 branches of my fork. Branding has changed back to neutral and everything appears to be working good, but if @KevinJDurant wouldn't mind checking the search torrent feature that would be handy. Searching for torrents works fine on the live demo but not on XAMPP (they are the same files) and I'm not sure whats up with that.

Edit: 
tested the search feature with @KevinJDurant 's fork on XAMPP and it didnt work so maybe its just XAMPP.

Edit 2:
Repulled my forks and uploaded to live demo; search feature is fine.. Good to pull if you like.

Branches
------------
Master: Neutral branding so you are able to pull at your convenience.
OpenTorrents: Live setup from the demo site @ http://torrents.azukachan.com


Changes
------------
1)Users can delete torrents from my-torrents.php
2)Branding is now neutral
3)Delete torrent message changed.
4)Scraper.php updated
5)Remove < code > from upload form for security reasons.
6)Users now login with username instead of email address.

AzukaChan committed:
Users can delete torrents

Torrents can be deleted from my-torrents.php

AzukaChan committed:
Neutral Branding

AzukaChan committed:
Added 'news' page

AzukaChan committed:
Update README.md

AzukaChan committed:
Update README.md

AzukaChan committed:
Login with Username

Previously users would have to login with email address.

AzukaChan committed:
Username login

AzukaChan committed:
Update index.php

accommodate for news.php

AzukaChan committed:
Update delete_torrent_file.php

AzukaChan committed:
Remove < code > from upload form

Remove < code > from upload form for security reasons.

AzukaChan committed:
Update scraper.php

AzukaChan committed:
Update README.md

AzukaChan committed:
Update README.md

AzukaChan committed:
Update search.php

Category change

AzukaChan committed:
Update search.php

Category change

AzukaChan committed:
Update my-torrents.php

delete confirmation message change.

KevinJDurant commented on en/view/my-torrents.php:
```diff
@@ -187,7 +187,7 @@
                             <td data-label="Download">
                                 <a href="'.$row["magnet"].'"><span class="glyphicon glyphicon-magnet link"></span></a>
-                                <a href="delete_torrent_file.php?delete_id='.$row["torrents_id"].'"  class="delete" data-confirm="Are you sure to delete this torrent file?"><span class="glyphicon glyphicon-trash link"></span></a>
+                                <a href="delete_torrent_file.php?delete_id='.$row["torrents_id"].'"  class="delete" data-confirm="Are you sure you want to permanently remove this torrent from the databse?"><span class="glyphicon glyphicon-trash link"></span></a>
```
```suggestion
                                <a href="delete_torrent_file.php?delete_id='.$row["torrents_id"].'"  class="delete" data-confirm="Are you sure you want to permanently remove this torrent from the database?"><span class="glyphicon glyphicon-trash link"></span></a>
```

KevinJDurant commented on en/account/login.php:
```diff
@@ -1,188 +1,188 @@
+    <!-- Standard Meta -->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="OpenTorrents: Free Software, Games and Music!">
```
```suggestion
    <meta name="description" content="OpenTorrentSite: Free Software, Games and Music!">
```

KevinJDurant commented on en/account/login.php:
```diff
@@ -1,188 +1,188 @@
+    <meta name="author" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Login | OpenTorrents</title>
```
```suggestion
    <title>Login | OpenTorrentSite</title>
```

KevinJDurant commented on en/account/login.php:
```diff
@@ -1,188 +1,188 @@
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="../../index.php">OpenTorrents</a>
```
```suggestion
                <a class="navbar-brand" href="../../index.php">OpenTorrentSite</a>
```

KevinJDurant commented on en/account/login.php:
```diff
@@ -1,188 +1,188 @@
+        <footer>
+            <div class="row">
+                <div class="col-lg-12">
+                    <p>Happy Downloading ~</p>
```
```suggestion
                    <p>Copyright &copy; Your Website 2017</p>
```

KevinJDurant commented:
@AzukaChan I have reviewed the code and suggested some changes. I can't commit my review to your branch, I'll merge once these small issues have been fixed.

KevinJDurant committed:
Update en/view/my-torrents.php

Thanks for noticing my rushed typing and spelling errors :D

Co-Authored-By: AzukaChan <42724187+AzukaChan@users.noreply.github.com>

KevinJDurant committed:
Update en/account/login.php

My bad, must of screwed up the neutral branding when I pulled from the live demo.

Co-Authored-By: AzukaChan <42724187+AzukaChan@users.noreply.github.com>

KevinJDurant committed:
Update en/account/login.php

Co-Authored-By: AzukaChan <42724187+AzukaChan@users.noreply.github.com>

KevinJDurant committed:
Update en/account/login.php

Co-Authored-By: AzukaChan <42724187+AzukaChan@users.noreply.github.com>

KevinJDurant committed:
Update en/account/login.php

Co-Authored-By: AzukaChan <42724187+AzukaChan@users.noreply.github.com>

AzukaChan commented:
@KevinJDurant 
My bad for the small issues.
Suggestions have been added.

AzukaChan committed:
Update my-torrents.php

Forgot to include "Books" in category dropdrown-menu.

KevinJDurant commented:
@AzukaChan I have squashed your changes into 1 commit and merged them into the master branch. Thanks for your contributions!
